docs/patterns/components/Address/index.js

```diff
@@ -16,6 +16,9 @@ class AddressDemo extends Component {
       <div className="data-list-demo">
         <Demo
           component={Address}
+          componentDescription={
+            'The address element indicates contact information for people or organizations. It may include any type of contact information including a physical address.'
```

Comment:
This could read as "The Address component..." similar to how you have "The Workspace component".